### PR TITLE
🐛 Source Hubspot: Fix non exists updated at field error

### LINKED ISSUE #11306: Hubspot KeyError
## Current Behavior
Key error, see logs
[logs-3856.txt](https://github.com/airbytehq/airbyte/files/8321399/logs-3856.txt)

## Expected Behavior
The error doesn't happen

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
airbytehq/airbyte#11306 - Source Hubspot: Fix  non exists updated at field error

## How
Updated state setter if `updatedAt` field non exist will be used `start_date` value.

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

### CODE REVIEW

**Review by bazarnov — APPROVED** [review]
Please check the documentation, pr and link.

**Review by annalvova05 — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py [hunk 1]
```diff
@@ -667,7 +667,7 @@ def state(self) -> Optional[Mapping[str, Any]]:
 
     @state.setter
     def state(self, value):
-        state_value = value[self.updated_at_field]
+        state_value = value.get(self.updated_at_field, self._state)
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py:670]: we should create an issue to refactor state management in this connector. It's very confusing that we have: 

`get_updated_state`
`_update_state`
`@state.setter`
`_start_date` and `_state` always set to the same thing

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py (hunk 1 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py:670]: Created https://github.com/airbytehq/airbyte/issues/11392

**Pull request merged**